tools/react-integration-tester/src/files/jest.config.js.template

```diff
@@ -85,3 +86,28 @@ function getTsPathAliases(){
 
   return tsPathAliases;
 }
+
+/**
+ * @param {import('@jest/types').Config.InitialOptions} config - project origin provided config
+ * @param {import('@jest/types').Config.InitialOptions} workspaceConfig - workspace preset config
+ */
+function createOriginProjectConfig(config, workspaceConfig) {
+  // no preset property, we will use configuration as is
```

Comment:
not every project has `preset` set, this creates resolved jest config based on origin setup to make it repo agnostic